### PR TITLE
chore: change app name to correct one

### LINKED ISSUE #5: change application name
Change the app name in package.json, manifest.json and other places.

### PR DESCRIPTION
**Pull request by jashpam**
Also removed changelog.

closes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+  "name": "graasp-app-badges",
+  "version": "0.0.0",
   "private": true,
-  "author": "React EPFL",
+  "author": "Pamela Andrade Sevillano",
   "contributors": [
-    "Juan Carlos Farah",
-    "Hassan Abdul Ghaffar",
-    "Mashkour Abdel Aziz"
+    "Juan Carlos Farah"
```
Comment (juancarlosfarah) [inline package.json:8]: You should add André as well.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Looking good! But maybe also add André to the contributor's list.

File: package.json (hunk 1 above)
Comment (jashpam) [reply package.json:8]: OK. Will do.

**Comment by jashpam** [comment]
@juancarlosfarah I did the changes you requested.

**Review by juancarlosfarah — APPROVED** [review]

**Comment by juancarlosfarah** [comment]
@jashpam, you can merge when you're ready.

**Pull request merged**